src/editor/Renderer.cpp

```diff
@@ -1331,26 +1400,21 @@ void Renderer::pickObject() {
 		pickInfo.entIdx = -1;
 		gui->showLightmapEditorUpdate = true;
 
-		if (pickInfo.modelIdx >= 0 && pickInfo.faceIdx >= 0) {			
-			if (selectedFaces.size() && selectMapIdx != pickInfo.mapIdx) {
-				logf("Can't select faces across multiple maps\n");
-			}
-			else {
-				selectMapIdx = pickInfo.mapIdx;
-				bool select = true;
-				for (int i = 0; i < selectedFaces.size(); i++) {
-					if (selectedFaces[i] == pickInfo.faceIdx) {
-						select = false;
-						selectedFaces.erase(selectedFaces.begin() + i);
-						break;
-					}
+		if (pickInfo.modelIdx >= 0 && pickInfo.faceIdx >= 0) {		
+			bool select = true;
+			for (int i = 0; i < selectedFaces.size(); i++) {
+				if (selectedFaces[i] == pickInfo.faceIdx) {
+					select = false;
+					selectedFaces.erase(selectedFaces.begin() + i);
+					break;
 				}
+			}
 
-				mapRenderers[pickInfo.mapIdx]->highlightFace(pickInfo.faceIdx, select);
+			g_app->getSelectedMap()->highlightFace(pickInfo.faceIdx, select);
 
-				if (select)
-					selectedFaces.push_back(pickInfo.faceIdx);
-			}
+			if (select)
+				selectedFaces.push_back(pickInfo.faceIdx);
+			
 		}

```

Comment:
There's no multi-map editing yet, but I still want this there in preparation for that.

Reply:
@wootguy i not see problem in these code